java/sage/Wizard.java

```diff
@@ -3156,6 +3177,11 @@ public SeriesInfo addSeriesInfo(int legacySeriesID, int showcardID, String title
         series.seriesImages = Pooler.EMPTY_INT_ARRAY;
       else
         series.seriesImages = seriesImages.clone();
+      if (imageURLs == null || imageURLs.length == 0)
+        series.imageURLs = Pooler.EMPTY_2D_BYTE_ARRAY;
+      else
+        series.imageURLs = imageURLs;
+      series.imageURLs = (imageURLs == null) ? Pooler.EMPTY_2D_BYTE_ARRAY : imageURLs.clone();
```

Comment:
Good point. Fixed it.

I tried my best to emulate whatever you were doing for the particular object I was working on.